src/Task/tasks.yml

```diff
@@ -30,6 +30,7 @@ Wunderio\GrumPHP\Task\CheckFilePermissions\CheckFilePermissionsTask:
         - '/node_modules/'
         - '/core/'
         - '/libraries/'
+        - '/web/modules/contrib/'
```

Comment:
This assumes that there is web. Should we just say modules/contib?

Reply:
Thanks, yes, it was a bad assumption. 70acc03 fixes.